docs/SONATA_DEVELOPER_GUIDE.md

```diff
@@ -573,6 +573,19 @@ For `biophysical` models defined according to the *bmtk* (see above), the namesp
 
 For `biophysical` models defined according to the *hoc* (see above), the namespace definition is to be filled in by the Allen folks.
 
+#### Nodes - Enum datatypes
+
+It is often the case that attributes storing strings have a limited number of possible values (for instance, `model_type`, or cell morphological type).
```

Comment:
"attributes storing strings" -> "string attributes" or "attributes of string type"

Reply:
Done.